### PR TITLE
Added the blocked request full url on block page and ABR json the block page & added new block page implementation 

### PR DESCRIPTION
**Pull request by chen-zimmer-px**
…cked url

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # Change Log
+
+## [v6.9.0](https://github.com/PerimeterX/perimeterx-java-sdk/compare/6.8.2...HEAD) (2023-XX-XX)
+- Added new block page implementation
```
Comment (etrpx) [inline CHANGELOG.md:4]: These two are confusing and not accurate, you didn't add a new block page.
You added a field to ABR and captcha template.
So maybe something like:
`Added blocked URL to ABR and captcha template` or `Updated captcha template and ABR with blocked URL`

**Review by etrpx — COMMENTED** [review]
Overall looks good, see small comment on the changelog.
Which E2E tests did you check?
Did you update them as well?

File: src/main/java/com/perimeterx/api/blockhandler/DefaultBlockHandler.java [hunk 2]
```diff
@@ -77,14 +77,20 @@ protected void sendMessage(String blockPageResponse, HttpServletResponseWrapper
                 responseWrapper.setContentType(Constants.CONTENT_TYPE_APPLICATION_JSON);
 
                 Map<String, String> props = TemplateFactory.getProps(context, pxConfig);
+                final String blockedUrlProperty = props.get("blockedUrl");
+                String blockedUrl = "";
+                if (blockedUrlProperty != null) {
```
Comment (etrpx) [inline src/main/java/com/perimeterx/api/blockhandler/DefaultBlockHandler.java:82]:
```suggestion
                final String blockedUrl = blockedUrlProperty != null ? "&b=" + Base64.encodeToString(props.get("blockedUrl").getBytes(), false) : "";

```

**Review by etrpx — APPROVED** [review]
Approved and added a suggestion

File: CHANGELOG.md (hunk 1 above)
Comment (chen-zimmer-px) [reply CHANGELOG.md]: done

**Review by etrpx — APPROVED** [review]

**Pull request merged**